src/data/scripts/hullmods/vayra_modular_shields.java

```diff
@@ -398,7 +389,7 @@ private void doShieldEmitterStuff(ShipAPI ship) {
         }
     }
 
-    private void triggerShield(ShipAPI ship, ShipAPI source, Vector2f point, Float damage, DamageType type, ShipAPI generator, Color color, Boolean hard) {
+    private static void triggerShield(ShipAPI ship, ShipAPI source, Vector2f point, Float damage, DamageType type, ShipAPI generator, Color color, Boolean hard) {
```

Comment:
why is this static? remember ,static methods cause memory leaks[*]. lets not have static methods if we can. and we surely somehow can. lets deal with that problem when we get to it. please remove `static`

[*] - static methods don't cause leaks, but they make deducing when to GC something harder, and thus waste more memory than necessary or prevent this object from being GC'd at all, thus effectivelly causing a memory leak.